data/mods/Xedra_Evolved/mutations/temporary.json

```diff
@@ -29,6 +29,79 @@
     "condition": "ALWAYS",
     "values": [ { "value": "MAX_HP", "multiply": 0.4 }, { "value": "STRENGTH", "add": 6 } ]
   },
+  {
+    "type": "mutation",
+    "id": "GOBLIN_FRUIT_MUSIC",
+    "name": { "str": "The fruit's song" },
+    "//": "Until the day EOCs can spawn equiped items, the fruit will give a mutation that holds the music player.",
+    "valid": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "player_display": false,
+    "points": 2,
+    "visibility": 0,
+    "ugliness": 0,
+    "description": { "str": "Gives the integrated music player.  Should not be player-visible.", "//~": "NO_I18N" },
```

Comment:
I really like this effect.  It's very on point.  I just wish we could dictate music types it plays.

Reply:
That could be done with a musical_instrument tick_action, but that would flood the player with messages for hours, which wouldn't fit the theme of "a song is struck in your head. Luckily, it's not an annoying song."

I intentionally left it vague as it's always an enjoyable song, even to those who hate music.